packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -0,0 +1,189 @@
+import atexit
+import logging
+import os
+import signal
+from subprocess import TimeoutExpired
+from typing import Callable, List, Optional, Union, Literal, Tuple
+
+import flet
+from flet import Control, Page, ScrollMode
+from flet.utils import is_windows
+
+from RPA.Assistant.types import Result, Location
+
+
+def resolve_absolute_position(
+    location: Union[Location, Tuple],
+) -> Tuple[int, int]:
+
+    if location is Location.TopLeft:
+        return (0, 0)
+    elif isinstance(location, tuple):
+        return location
+    else:
+        raise ValueError(f"Invalid location {location}")
+
+
+class FletEvent:
+    target: str
+    name: str
+    data: str
+    control: Control
+    page: Page
+
+class TimeoutException(RuntimeError):
+    """Timeout while waiting for dialog to finish."""
+
+class FletClient:
+    """Class for wrapping flet operations"""
+
+    def __init__(self) -> None:
+        self.results: Result = {}
+        self.page: Optional[Page] = None
+
+        self._conn = self._preload_flet()
+        self._elements: List[List[Control]] = [[]]
+        self._invisible_elements: List[List[Control]] = [[]]
+        self._pagination = 0
+        self._fvp = None
+        atexit.register(self._cleanup)
+
+    @property
+    def current_elements(self):
+        return self._elements[self._pagination]
+
+    @property
+    def current_invisible_elements(self):
+        return self._invisible_elements[self._pagination]
+
+    def _cleanup(self) -> None:
+        # Source: https://github.com/flet-dev/flet/blob/89364edec81f0f9591a37bdba5f704215badb0d3/sdk/python/flet/flet.py#L146
+        self._conn.close()
+        if self._fvp is not None and not is_windows():
+            try:
+                logging.debug(f"Flet View process {self._fvp.pid}")
+                os.kill(self._fvp.pid + 1, signal.SIGKILL)
```

Comment:
Feedback from Jippo and Fabio: This is actually pretty dangerous and may kill a random process. If we want to kill a process and all it's children LSP had some code we could take inspiration from here: https://github.com/robocorp/robotframework-lsp/blob/f485ec493109311f32df08a7e0b2e243ed974d5c/robocorp-python-ls-core/src/robocorp_ls_core/basic.py#L220